dev/spacelift/dpe-sandbox/main.tf

```diff
@@ -118,15 +128,15 @@ resource "spacelift_stack_destructor" "k8s-stack-destructor" {
 
 resource "spacelift_aws_integration_attachment" "k8s-aws-integration-attachment" {
   # org-sagebase-dnt-dev-aws-integration
-  integration_id = "01J3DNYVM4AWWSDY3QEVRMQ076"
+  integration_id = "01J3R9GX6DC09QV7NV872DDYR3"
```

Comment:
Is this another thing that should be parameterized at some point (especially if we ever have both `dev` and `prod` deployments)?

Reply:
I added these changes to https://github.com/Sage-Bionetworks-Workflows/eks-stack/pull/15